### ISSUE TITLE
pwsh command-line help errors

### ISSUE THREAD

**Issue by sdwheeler** [PowerShell/PowerShell#9948]
# Steps to reproduce

```powershell
pwsh -?
```

# Expected behavior

# Actual behavior

There are several issues in the output of help for the command-line options.

1. The `-SettingsFile` parameter is not listed in the syntax block but it is listed and described below.

2. Was `-LoadProfile` actually added? For what purpose? It is not listed in the output for PSv6.

3. The description for `-WorkingDirectory` includes markdown formatting for the "Set-Location" example, which is not appropriate for console output.

```
-WorkingDirectory | -wd
    Sets the working directory at the start of PowerShell given a valid PowerShell directory path.
    Executing `Set-Location -LiteralPath <path>` at startup.
```

4. The the documentation in about_pwsh.md has been improved with more information and better grammar. We should sync those changes into the code.

# Environment data

```none
Name                           Value
----                           -----
PSVersion                      7.0.0-preview.1
PSEdition                      Core
GitCommitId                    7.0.0-preview.1
OS                             Microsoft Windows 10.0.18362
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by iSazonov** [comment]
LoadProfile RFC is still not approved.

**Comment by sdwheeler** [comment]
LoadProfile is currently in the preview build but will be changing. I fixed the documentation problems in #9952. This issue can be closed when #9952 is merged.
